### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by axlwaii**
First version for our contributing guidelines.

### CODE REVIEW

**Comment by LostCrew** [comment]
Closes #30

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Contributing
+We always welcome pull requests. This guide will help you to ensure that that your time is spent wisely.
```
Comment (LostCrew) [inline CONTRIBUTING.md:2]:
```suggestion
## Contributing

We always welcome pull requests. This guide will help you to ensure that that your time is spent wisely. 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## Contributing
+We always welcome pull requests. This guide will help you to ensure that that your time is spent wisely. 
+
+If you are thinking of introduing a new feature or making a significant change to an existing behavior, discuss it
+first with the maintainers by creating an issue.
+
+### Process
+- Fork and clone the repo
```
Comment (LostCrew) [inline CONTRIBUTING.md:8]:
```suggestion
### Process

- Fork and clone the repo
```

**Review by jakubfiala — APPROVED** [review]
looks good to me! thank you

**Pull request merged**